circuit-benchmarks/Cargo.toml

```diff
@@ -9,7 +9,7 @@ license = "MIT OR Apache-2.0"
 [dependencies]
 halo2_proofs = { git = "https://github.com/privacy-scaling-explorations/halo2.git", tag = "v2022_09_10" }
 ark-std = { version = "0.3", features = ["print-trace"] }
-zkevm-circuits = { path = "../zkevm-circuits" }
+zkevm-circuits = { path = "../zkevm-circuits", features = ["test"] }
```

Comment:
Is this still needed?

Reply:
Yes, because the fn rand_tx comes from the zkevm_circuit::test_utils which is enabled by feature `test`.

Reply:
Hmmm fair enough, let's see what the EF guys think. :)